### PR TITLE
CAL-270 updated documentation templates to include introduction sections

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

adds intro sections to templated documentation

#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth @clockard 

#### Choose 2 committers to review/merge the PR.

@coyotesqrl
@jlcsmith
@lessarderic
@pklinef
@shaundmorris

#### How should this be tested?

verify intended sections are being included. See attached pdf.

#### Any background context you want to provide?

`-contents.adoc` files unchanged except added headers, content review optional.

#### What are the relevant tickets?

[CAL-270](https://codice.atlassian.net/browse/CAL-270)

#### Screenshots (if appropriate)

[jdocumentation.pdf](https://github.com/codice/alliance/files/1006145/jdocumentation.pdf)

#### Checklist:
- [x] Documentation Updated
	- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -79,6 +79,7 @@
         <ddf-branding>DDF</ddf-branding>
         <ddf-branding-lowercase>ddf</ddf-branding-lowercase>
         <ddf-branding-expanded>Distributed Data Framework</ddf-branding-expanded>
+        <cal-branding-expanded>Codice Alliance</cal-branding-expanded>
```
Comment (shaundmorris) [inline pom.xml]: Do we have any general guidance on when to use`branding- expanded` vs just `branding`?

Comment (ricklarsen) [inline pom.xml]: Nothing formal. I guess that's a branding decision ultimately. I'm not 100% sure it's necessary to say "Codice Alliance" every time, but that's how it was in this intro section. Afterward, it's always just "Alliance"

**Comment by ricklarsen** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**